test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+source settings.sh
+DEVICE=ltsptest01
+
+help() {
+    cat <<-_EOF
+    
+test.sh
+This script creates virtual network adapter which can be used for bridging
+
+Usage:
+$0 [options]
+
+Options:
+    start   This option will create virtual adpater
+    stop    This option will destory virtual adpater
+    
+_EOF
+}
+
+if [ $UID != 0 ]; then
```

Comment:
Also good practice is to always use `[[` and `]]` for **if** instead of single brackets. Find out what is the difference between them and why the double brackets are recommended (look it up on the book or find something on internet).